### PR TITLE
Create new database for each run of integration tests

### PR DESCRIPTION
**Pull request by sfc-gh-astus**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added new automated tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [x] I've described my changes in the section below.

### Changes description
Creating new database before integration tests
Cleanup database after tests

### CODE REVIEW

File: tests_integration/snowflake_connector.py [hunk 1]
```diff
@@ -10,19 +11,41 @@
...
+    config = {
+        "application": "INTEGRATION_TEST",
+        "account": _get_from_env("ACCOUNT", True),
+        "user": _get_from_env("USER", True),
+        "password": _get_from_env("PASSWORD", True),
+        "host": _get_from_env("HOST", True),
+        "role": _get_from_env("ROLE", True),
+    }
+    config = {k: v for k, v in config.items() if v is not None}
+    connection = connector.connect(**config)
```
Comment (sfc-gh-astus) [inline tests_integration/snowflake_connector.py:26]: Maybe someone will have better idea

File: tests_integration/snowflake_connector.py [hunk 2]
```diff
@@ -10,19 +11,41 @@
...
+
+    connection.execute_string(f"create database {database_name}")
```
Comment (sfc-gh-turbaszek) [inline tests_integration/snowflake_connector.py:28]: Why not `execute`?

File: tests_integration/snowflake_connector.py [hunk 3]
```diff
@@ -10,19 +11,41 @@
...
+    os.environ[f"{_ENV_PARAMETER_PREFIX}DATABASE"] = database_name
+
+    yield database_name
+
+    connection.execute_string(f"drop database {database_name}")
+    del os.environ[f"{_ENV_PARAMETER_PREFIX}DATABASE"]
```
Comment (sfc-gh-turbaszek) [inline tests_integration/snowflake_connector.py:34]:
```suggestion
    del os.environ[f"{_ENV_PARAMETER_PREFIX}_DATABASE"]
```
Imho this is more readable than using `_` suffix in the prefix

File: tests_integration/snowflake_connector.py (hunk 2 above)
Comment (sfc-gh-astus) [reply tests_integration/snowflake_connector.py:28]: SnowflakeConnection don't have `execute` method

File: tests_integration/snowflake_connector.py (hunk 3 above)
Comment (sfc-gh-astus) [reply tests_integration/snowflake_connector.py]: Changed

**Review by sfc-gh-turbaszek — APPROVED** [review]

**Pull request merged**